DISCOVER/09_participant_selection.md

```diff
@@ -58,8 +58,9 @@
 **Further reading:**
 
 - [Would I attend my own conference?](http://radar.oreilly.com/2011/03/would-i-attend-my-own-conferen.html)
-- [Getting women to speak at your conference is hard. So?](http://www.dogsandshoes.com/2012/06/getting-women-and-other-under-represented-people-to-speak-at-your-conference-is-hard-so.html)
-- [A Better Call for Proposals](http://2016.cssconf.com.au/2016/07/01/cfp.html)
+<!---  ACCESS DENIED
+ [Getting women to speak at your conference is hard. So?](http://www.dogsandshoes.com/2012/06/getting-women-and-other-under-represented-people-to-speak-at-your-conference-is-hard-so.html)
+- [A Better Call for Proposals](http://2016.cssconf.com.au/2016/07/01/cfp.html)-->
```

Comment:
link moved to: https://web.archive.org/web/20211123193159/http://2016.cssconf.com.au/2016/07/01/cfp.html